### ISSUE TITLE
No loads

### ISSUE THREAD

**Issue by iLenTheme** [renettarenula/Tako#4]
Hello, I have over 2000 post but for a while when I want to move a comment that does not already loaded, it is the icon of loading and never finishes loading. That might be going ?, will not be for the new version of Wordpress? Please check this news

**Comment by jphorn** [comment]
Same issue. We have over 30.000 posts (10+ year old news site) and the posts dropdown never populates due to timeout issues. Could you please rethink the logic to maybe check for large number of posts (1000+) and not include the dropdown, but let users just type in the post_id. This would be a very easy solution which helps small and large sites.

**Comment by iLenTheme** [comment]
Yes, there are many practical solutions to avoid overloading the list.
